tests/forte/data/extractor/base_extractor_test.py

```diff
@@ -0,0 +1,37 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import unittest
+
+from ft.onto.base_ontology import Token
+from forte.data.extractor.base_extractor import BaseExtractor
+
+
+class BaseExtractorTest(unittest.TestCase):
+
+    def test_BaseExtractor(self):
```

Comment:
How about we have consistent function name style? Like `text_base_extractor`?

Reply:
Fixed.